tck/src/main/java/org/reactivestreams/tck/SubscriberBlackboxVerification.java

```diff
@@ -287,7 +287,16 @@ public void required_spec209_blackbox_mustBePreparedToReceiveAnOnCompleteSignalW
       public void run(BlackboxTestStage stage) throws Throwable {
         final Publisher<T> pub = new Publisher<T>() {
           @Override
-          public void subscribe(Subscriber<? super T> s) {
+          public void subscribe(final Subscriber<? super T> s) {
+            s.onSubscribe(new Subscription() {
```

Comment:
Please be more helpful, very tired;
The TCK contains plenty "edgy" things, since they're designed to test a single thing only